Cordova.podspec

```diff
@@ -0,0 +1,34 @@
+Pod::Spec.new do |s|
+  s.name         = "Cordova"
```

Comment:
Added podspec for `Cordova` here. Will send an upstream PR to them as well. Once they add it and we upgrade to that release, we can get rid of this.